File: VB/WizardCustomizationQueryPage/Form1.vb
Imports DevExpress.DataAccess
Imports DevExpress.DataAccess.ConnectionParameters
Imports DevExpress.XtraBars.Ribbon
Imports System
Imports System.Collections.Generic
Imports System.ComponentModel
Imports System.Data
Imports System.Drawing
Imports System.Linq
Imports System.Text
Imports System.Threading.Tasks
Imports System.Windows.Forms

Namespace WizardCustomizationQueryPage
    Partial Public Class Form1
        Inherits RibbonForm

        Public Sub New()
            InitializeComponent()
            dashboardDesigner1.DataSourceWizardCustomization = New DataSourceWizardCustomization()
            Dim cpNorthwind As New MsSqlConnectionParameters() With {.ServerName = "localhost", .AuthorizationType = MsSqlAuthorizationType.Windows, .DatabaseName = "Northwind"}
            Dim cpAdventureWorks As New MsSqlConnectionParameters() With {.ServerName = "localhost", .AuthorizationType = MsSqlAuthorizationType.Windows, .DatabaseName = "AdventureWorksDW2012"}
            Dim cpContosoRetail As New MsSqlConnectionParameters() With {.ServerName = "localhost", .AuthorizationType = MsSqlAuthorizationType.Windows, .DatabaseName = "ContosoRetailDW"}
            dashboardDesigner1.CustomDataConnections.Add(New DataConnection("Local Northwind Connection", cpNorthwind))
            dashboardDesigner1.CustomDataConnections.Add(New DataConnection("Local AdventureWorks Connection", cpAdventureWorks))
            dashboardDesigner1.CustomDataConnections.Add(New DataConnection("Local ContosoRetail Connection", cpContosoRetail))
        End Sub
    End Class
End Namespace

File: CS/WizardCustomizationQueryPage/Form1.cs
using DevExpress.DataAccess;
using DevExpress.DataAccess.ConnectionParameters;
using DevExpress.XtraBars.Ribbon;
using System;
using System.Collections.Generic;
using System.ComponentModel;
using System.Data;
using System.Drawing;
using System.Linq;
using System.Text;
using System.Threading.Tasks;
using System.Windows.Forms;

namespace WizardCustomizationQueryPage {
    public partial class Form1 : RibbonForm {
        public Form1() {
            InitializeComponent();
            dashboardDesigner1.DataSourceWizardCustomization = new DataSourceWizardCustomization();
            MsSqlConnectionParameters cpNorthwind = new MsSqlConnectionParameters()
            {
                ServerName = "localhost",
                AuthorizationType = MsSqlAuthorizationType.Windows,
                DatabaseName = "Northwind"
            };
            MsSqlConnectionParameters cpAdventureWorks = new MsSqlConnectionParameters()
            {
                ServerName = "localhost",
                AuthorizationType = MsSqlAuthorizationType.Windows,
                DatabaseName = "AdventureWorksDW2012"
            };
            MsSqlConnectionParameters cpContosoRetail = new MsSqlConnectionParameters()
            {
                ServerName = "localhost",
                AuthorizationType = MsSqlAuthorizationType.Windows,
                DatabaseName = "ContosoRetailDW"
            };
            dashboardDesigner1.CustomDataConnections.Add(new DataConnection("Local Northwind Connection", cpNorthwind));
            dashboardDesigner1.CustomDataConnections.Add(new DataConnection("Local AdventureWorks Connection", cpAdventureWorks));
            dashboardDesigner1.CustomDataConnections.Add(new DataConnection("Local ContosoRetail Connection", cpContosoRetail));
        }
    }
}
